addons/binding/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/handler/DeviceHandler.java

```diff
@@ -179,6 +180,10 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                 if (command instanceof DecimalType) {
                     BigDecimal temperature = new BigDecimal(command.toString());
                     fritzBox.setSetTemp(ain, HeatingModel.fromCelsius(temperature));
+                } else if (command instanceof IncreaseDecreaseType) {
+                    BigDecimal settemp = (BigDecimal) getThing().getConfiguration().get(THING_SETTEMP);
+                    fritzBox.setSetTemp(ain, HeatingModel.fromCelsius(command.equals(IncreaseDecreaseType.INCREASE)
```

Comment:
The value of the configuration is updated from the polling job with a little delay. It is only temporarily fix. But yes, sending INCREASE or DECREASE multiple times results in the same temperature. If I write the new value back to the configuration, it will inc/dec with every command. I can change that.